src/Cms/Language.php

```diff
@@ -145,8 +146,13 @@ public function code(): string
 	 */
 	public static function create(array $props): static
 	{
+		$kirby = App::instance();
+
+		if ($kirby->user()?->role()->permissions()->for('languages', 'create') === false) {
```

Comment:
Is this correct if there's no authenticated user? I would assume that `$kirby->user()?` returns null here, which would then not match `false` and could bypass the permssion check.

Reply:
That's right @bastianallgeier . That's why I wrote for this case in `Additional context` part of the PR description :)

Btw currently there is already no check for the creating and deleting languages. I kept this behavior and that's why why I need your suggestion. I I remove the `null` check, that means breaking change. 

So what do you think?

Reply:
Sorry for completely missing this in your summary. You explained it perfectly there. I really think that this is something we need to fix in this case more strictly and definitely ask for an authenticated user. It's indeed a breaking change, but one that I think is necessary. Let's take the discussion about it to Discord.